src/app/loadout/mod-utils.ts

```diff
@@ -384,22 +388,25 @@ function createPluggingStrategy(
       }
     }
 
-    // If a destination socket couldn't be found for this plug, something is seriously wrong
+    // If a destination socket couldn't be found for this plug, something is seriously? wrong
```

Comment:
Might be worth calling out that something is wrong because we have a mismatch of how the assignment algorithm is assessing if a mod can be plugged into an item. At this point everything should be able to socket into something in an item

Reply:
well we `throw new Error(` not sure how much more we can call it out than that?

Reply:
Sorry I meant putting why it shouldn't happen in the comment. Not a big deal either way though